Modules/Server/Sources/PocketCastsServer/Public/Search/EpisodeSearchTask.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+public struct EpisodeSearchEnvelope: Decodable {
+    public let episodes: [EpisodeSearchResult]
+}
+
+public struct EpisodeSearchResult: Decodable {
+    public let uuid: String
+    public let title: String
+    public let publishedDate: Date
+    public let duration: Double?
+    public let podcastUuid: String
+    public let podcastTitle: String
+}
+
+public class EpisodeSearchTask {
+    var session = URLSession.shared
+
+    public init() {}
```

Comment:
Wdyt about allowing injecting of the URLSession?

```suggestion
    private let session: URLSession

    public init(session: URLSession = .shared) {
        self.session = session
    }
```